modules/mw.cx.MachineTranslationService.js

```
@@ -189,75 +189,69 @@ mw.cx.MachineTranslationService.prototype.cxtoken_error = function () {
+				console.log('Fetch failed:', data);
+			} else {
+				console.log('Fetch successful:', data);
+			}
+		})
+		.catch(error => {
+			console.error('Error fetching mdwiki token:', error);
+		})
+	return result;
+};
+
+mw.cx.MachineTranslationService.prototype.fetchCXServerToken_MDWIKI = function () {
```

ESLint

[error] Identifier 'fetchCXServerToken_MDWIKI' is not in camel case.

(camelcase)